akka-http-core/src/main/scala/akka/http/impl/engine/client/pool/NewHostConnectionPool.scala

```diff
@@ -560,8 +563,8 @@ private[client] object NewHostConnectionPool {
           super.onDownstreamFinish()
         }
         override def postStop(): Unit = {
-          log.debug("Pool stopped")
           slots.foreach(_.shutdown())
+          log.warning(s"Pool stopped")
```

Comment:
Does this have to be a warning? Isn't it 'normal' for a pool to stop?

Reply:
Right, after the idle timeout e.g. Let's make it an info or even debug (should also be logged at the PoolInterfaceActor as well).